ballerina-shell/modules/shell-core/src/main/java/io/ballerina/shell/utils/StringUtils.java

```diff
@@ -162,4 +162,19 @@ public static String getErrorStringValue(Throwable error) {
         }
         return error.getMessage();
     }
+
+    /**
+     * Convert unicode character to ballerina unicode format
+     *
+     * @param string Input unicode containing string to convert.
+     * @return reformatted unicode string.
+     */
+    public static String convertUnicode(String string){
```

Comment:
Maybe simplify as following?

```java
    public static String convertUnicode(String string){
        int character = string.charAt(0);
        return "\\" + "u" + "{" + Integer.toHexString(character) + "}";
    }
```

Reply:
Updated in 3083628406bc711786e3b472071f89c46d21e09e